agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -116,11 +116,8 @@ fi
 if [ "$sysinfo" == "none" ]; then
 	exit 0
 fi
-if [ "$sysinfo" == "default" ]; then
-	sysinfo=${sysinfo_opts_default[*]}
-fi
-if [ "$sysinfo" == "all" ]; then
-	sysinfo=${sysinfo_opts[*]}
+if [ "$sysinfo" == "default" ] || [ "$sysinfo" == "all" ]; then
```

Comment:
Why are there three variables? sysinfo_default_opts, sysinfo_opts and sysinfo - what do they do?

Reply:
sysinfo is set with the parameters passed by the user, sysinfo_default_opts is an array of all possible options while sysinfo_opts contains comma separated values.

Reply:
Do you need sysinfo_opts at all? If you can drop it easily, I'd suggest you do so. If not, then it should be renamed.